index.html

```diff
@@ -2435,37 +2435,58 @@ <h2>Cryptographic Suites</h2>
 verification of the data.
         </li>
         <li>
-          The specification MUST define a <dfn>suite selection algorithm</dfn> that accepts a [=JSON
-          object=] |proof| and uses the properties described in [[[#proofs]]] to return either
-          failure or a [=VC cryptographic suite=] [[Infra]] [=struct=]. This algorithm SHOULD be
-          registered in [[[VC-SPECS]]]. A
-          <dfn class="export">VC cryptographic suite</dfn> [=struct=] has the following
-          [=struct/items=].
+The specification MUST define a <dfn>suite selection algorithm</dfn> that
+accepts a set of options ([=map=] |options|) and returns a
+[=cryptographic suite=] ([=struct=] |cryptosuite|). This algorithm SHOULD be
+registered in [[[VC-SPECS]]]. A cryptographic suite [=struct=] has the
```

Comment:
Just knit-picking. We may want to use a different term for "registered" to avoid misunderstandings. The VC-SPECS document explicitly says that this is not a registry, and using the term "registered" may lead the reader to believe that it is. Just to be careful...

Reply:
```suggestion
[=cryptographic suite=] ([=struct=] |cryptosuite|). This algorithm SHOULD be
listed in [[[VC-SPECS]]]. A cryptographic suite [=struct=] has the
```

Reply:
Yes, agreed, I accepted @TallTed's change where he changed "registered" -> "listed".